src/scripts/main.js

```diff
@@ -1,5 +1,48 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const spider = document.querySelector('.spider');
+const spiderParent = spider.parentElement;
+
+spiderParent.addEventListener('click', function(e) {
```

Comment:
Replace 'function(e)' with an arrow function to keep the code consistent: (e) => {}